opentelemetry-otlp/tests/integration_test/tests/logs.rs

```diff
@@ -115,6 +123,49 @@ mod logtests {
         Ok(())
     }
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+    #[cfg(any(feature = "tonic-client"))]
+    pub async fn logs_simple_tokio_multi_thread() -> Result<()> {
+        logs_simple_tokio_helper().await
+    }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    #[cfg(any(feature = "tonic-client"))]
+    pub async fn logs_simple_tokio_multi_with_one_worker() -> Result<()> {
+        logs_simple_tokio_helper().await
+    }
+
+    // Ignored, to be investigated
+    #[ignore]
+    #[tokio::test(flavor = "current_thread")]
+    #[cfg(any(feature = "tonic-client"))]
+    pub async fn logs_simple_tokio_current() -> Result<()> {
+        logs_simple_tokio_helper().await
+    }
+
+    async fn logs_simple_tokio_helper() -> Result<()> {
```

Comment:
nit - this can be shared across both simple/batch, by taking is_simple as bool arg.